descriptors/ModuleDescriptor-template.json

```diff
@@ -0,0 +1,108 @@
+{
+  "id": "${artifactId}-${version}",
+  "name": "Finance Funds",
+  "requires": [],
+  "provides": [
+    {
```

Comment:
Actually this story does not require defining any interface, there is another story https://issues.folio.org/browse/MODFUND-59. But I think you can leave this part of work as for it is already done, just fix following comments.